### ISSUE TITLE
Error on https://ruffle.rs/demo/

### ISSUE THREAD

**Issue by LollipopFt** [ruffle-rs/ruffle#10666]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:602:51
Error stack:
```
339/O/n.wbg.__wbg_new_15d3966e9981a196@https://ruffle.rs/demo/339.index.js:1:48008
core::ops::function::FnOnce::call_once{{vtable.shim}}::h31d10b9661c38166@:wasm-function[8139]:0x7ef234
std::panicking::rust_panic_with_hook::hc53aea0352e77326@:wasm-function[5466]:0x727ea2
core::panicking::panic_fmt::hf4a9df75710ece83@:wasm-function[8020]:0x7e9e72
core::panicking::panic::h54beb3f515dd3c6d@:wasm-function[7387]:0x7b5ff1
ruffle_core::bitmap::bitmap_data::BitmapData::update_dirty_texture::hefb37dc961254f68@:wasm-function[1233]:0x39a90c
<ruffle_core::library::MovieLibrarySource as ruffle_render::bitmap::BitmapSource>::bitmap_handle::h38d21e8290bf8b6c@:wasm-function[3216]:0x60d41d
<ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::register_shape::h20d8ef5807f9087b@:wasm-function[1035]:0x33c94b
ruffle_core::display_object::movie_clip::MovieClipData::define_shape::h8bcf323e115a57d3@:wasm-function[3030]:0x5f0362
ruffle_core::display_object::movie_clip::MovieClip::preload::h515cadf80469be7e@:wasm-function[619]:0x1c5cd7
ruffle_web::Ruffle::tick::h23b5ea28441fd7bc@:wasm-function[694]:0x23a296
ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::h8b0a8540a2d13a96@:wasm-function[9758]:0x8269da
<dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::h56085f3bbff8f367@:wasm-function[8652]:0x7fac21
A@https://ruffle.rs/demo/339.index.js:1:2588
f@https://ruffle.rs/demo/339.index.js:1:2265

```

# Player Info
Allows script access: false

# Page Info
Page URL: https://ruffle.rs/demo/
SWF URL: https://ruffle.rs/demo/swfs/alien_hominid.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:111.0) Gecko/20100101 Firefox/111.0
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-04-12
Channel: nightly
Built: 2023-04-12T00:25:02.383Z
Commit: da689b7687d6bb23f37251e902ccddabdfcc5f48
Is extension: false

# Metadata
width: 550
height: 400
frameRate: 24
numFrames: 24
swfVersion: 6
backgroundColor: #000000
isActionScript3: false
uncompressedLength: 2037879

**Comment by n0samu** [comment]
Hi, can you tell us what led up to this error? Did it happen at a specific point in the game?

**Comment by waspennator** [comment]
> Hi, can you tell us what led up to this error? Did it happen at a specific point in the game?

I can vouch for the user that it happens to me too, Alien Hominid is one of those swfs that on my S22 beforehand would render a black background when ingame on all chrominium browsers, but not on firefox.

I can confirm though that it doesn't happen on my laptop though.

**Comment by LollipopFt** [comment]
for me, once i clicked the swf file in the dropdown list, it instantly crashed.

**Comment by Lord-McSweeney** [comment]
Fixed by #10908.
